src/tests/ftest/rebuild/io_conf_run.yaml

```diff
@@ -0,0 +1,61 @@
+hosts:
+  test_servers:
+    - boro-A
+    - boro-B
+    - boro-C
+    - boro-D
+    - boro-E
+    - boro-F
+    - boro-G
+    - boro-H
+timeout: 900
+server_config:
+  name: daos_server
+  servers:
+    targets: 8
+    bdev_class: nvme
+    bdev_list: ["0000:5e:00.0","0000:5f:00.0"]
+pool:
+  scm_size: 14G
```

Comment:
If you are enabling 'nvme' above you will need to specify 'nvme_size' at pool create in order to take advantage of nvme.

Reply:
In this pool creation is not used based on Python class, it's similar how daos_tests uses.